server/server.ts

```diff
@@ -1,76 +1,26 @@
-import path from 'path'
-
-import { default as axios } from 'axios'
 import express from 'express'
 import cors from 'cors'
+import path from 'path'
 import compression from 'compression'
 import dotenv from 'dotenv'
-
-dotenv.config()
+import cookieParser from 'cookie-parser'
+import { NestFactory } from '@nestjs/core'
+import { AppModule } from './modules/app.module'
 
 const PORT = process.env.PORT ?? 5000
 
-const app = express()
-
-app.use(compression())
-app.use(express.json())
-app.use(cors({ origin: '*' }))
-
-app.use(express.static('client/build'))
-
-app.get('/token', async (_, res) => {
-    const body = {
-        client_id: process.env.Client_Id,
-        client_secret: process.env.Client_Secret,
-        scope: 'https://graph.microsoft.com/.default',
-        grant_type: 'client_credentials',
-    }
-
-    app.get('*', (_, res) => {
-        res.sendFile(path.join(__dirname, '../client/build/index.html'))
-    })
-
-    const formData = Object.entries(body)
-        .reduce((acc, [key, value]) => {
-            acc.push(`${key}=${encodeURIComponent(value)}`)
-            return acc
-        }, [])
-        .join('&')
-
-    const { data } = await axios.post(
-        'https://login.microsoftonline.com/a6dfed0e-808d-4a2e-ae4f-9adac874f50d/oauth2/v2.0/token',
-        formData,
-        { headers: { 'Content-Type': 'application/x-www-form-urlencoded' } }
-    )
-
-    res.cookie('access_token', data.access_token, {
-        httpOnly: true,
-        secure: process.env.NODE_ENV === 'production',
+dotenv.config()
+async function bootstrap() {
+    const app = await NestFactory.create(AppModule)
+    app.use(cookieParser())
+    app.use(compression())
+    app.use(express.json())
+    app.use(cors({ origin: '*' }))
+    app.use(express.static('client/build'))
+    app.use('*', express.static('client/build/index.html'))
```

Comment:
I don't this  * here would works correctly, I might take over all calls

This should work, in bottom of controller
   @Get('*')
    async home(@Res() response: Response) {
        response.sendFile(path.join(__dirname, '..', 'client', 'build', 'index.html'))
    }
   in the controller, I used it in the past

Reply:
It acts as fallback, 

@get('*')
async home(@res() response: Response) {
response.sendFile(path.join(__dirname, '..', 'client', 'build', 'index.html'))
}

did not work so i removed it

Reply:
I looked at you old commit you had it a bit differently, there was pass through enabled, and the path was setup differently. Try this as is, if it didn't work still, we can utilize this

Reply:
Worked like a charm 🗡️